app/assets/v2/js/cart.js

```diff
@@ -392,24 +395,19 @@ Vue.component('grants-cart', {
       const estimateZkSync = Number(this.zkSyncEstimatedGasCost); // zkSync gas cost estimate
       const estimatePolygon = Number(this.polygonEstimatedGasCost); // polygon gas cost estimate
```

Comment:
It looks like the gas estimations are still directly comparing gas used, and not the actual USD cost as described in item 2 of #9426. Though maybe I'm missing it somewhere?

Reply:
this feature hasn't be done yet... it'll be handled in a follow-up pr since it isn't polygon specific

Reply:
Got it, so this PR looks good to me then